assets/css/base/_base.scss

```diff
@@ -1 +1,52 @@
 // Base styles.
+
+// Typography
+h1 {
+	font-size: 4em ;
+	font-weight: semi-bold ;
+}
+h2 {
+	font-size: 3em ;
+	font-weight: medium ;
+}
+h3 {
+	font-size: 1.5em ;
+	font-weight: bold ;
+}
+p {
+	font-size: 1em ;
+	font-weight: regular ;
+}
+
+// Links and Buttons
+a {
+	color: #6895c6;
+	font-weight: bold;
+	text-decoration: none;
+}
+a:hover,
+a:focus {
+	background-color: #FADC90;
+}
+
+button {
+	background-color: #cee6c5;
+	border: 1px solid #8CAABB;
+	border-radius: 70px;
+	box-sizing: border-box;
+	transform: perspective(1px) translateZ(0);
+  }
+button:before {
```

Comment:
You should use a `box-shadow` for this instead of pseudo-content.

Reply:
I have tried. I am unable to duplicate the button styling of the Figma file with box-shadow. It allows for opaque shadows but there is no option to keep the border of the shadow present.

Reply:
Okay cool, pseudo-content works then.